src/Kernel/Magic/QirMagic.cs

```diff
@@ -25,19 +25,32 @@ public class QirMagic : AbstractMagic
             "qir",
             new Microsoft.Jupyter.Core.Documentation
             {
-                Summary = "TODO.",
+                Summary = "Compiles a given Q# entry point to QIR, then executes the program.",
```

Comment:
I don't think it actually executes, but returns the resultant QIR?

Reply:
Oh it _doesn't_ execute? It saves the QIR to a given file. Okay I'll make the changes for that.

Reply:
Okay, I've corrected the text.